File: Poo/Poo/Poo aulas/Aula 2/main.py
from Carro import *

if __name__ == "__main__":
    carro1 = Carro("Chevrolet", 2006, "Preto", "Corsa", "LTD22", "Manual")
    carro2 = Carro("FIAT", 2000, "Branco", "Uno com escada em cima", "ENGSOFT", "Manual")

    print(carro1)

    carro1.start()

    carro1.stop()

    print(carro2)

    carro2.start()

    carro2.drive(35)

    carro2.drive(100)

    carro2.stop()

File: Poo/Poo/Poo aulas/Aula 2/Main.java
public class Main {
    public static void main(String[] args) {
        // Criando o objeto Carro
        Carro meuCarro = new Carro("Fiat", "Uno com escada em cima", 2000);

        // Acessando os atributos do objeto meuCarro
        System.out.println("Carro "+ meuCarro.getMarca() + " "+ meuCarro.getModelo() + " " + meuCarro.getAno());
    }
}
